apps/src/aiComponentLibrary/chatMessage/ChatMessage.tsx

```diff
@@ -20,14 +23,20 @@ interface ChatMessageProps {
   messageStyle?: 'default' | 'warning' | 'danger';
 }
 
-/*
- * A rehype component map used to map between `pre` tags and `CopyableCodeBlock` components.
- *
- * For performance reasons, it is the `SafeMarkdown` consumer's responsibility to create the
- * rehypeMap outside  of the component function or to define the mapping in an ES module and
- * import it, if used in multiple components. See `SafeMarkdown` for more info.
- **/
-const rehypeMap = {pre: CopyableCodeBlock};
+const codeCopiedAnalytics = (isTA: boolean) => () =>
+  getStore().dispatch(sendAnalytics(EVENTS.CODE_COPIED, {isTA: !!isTA}));
```

Comment:
I realized the code I suggested still had `!!` but it shouldn't be necessary anymore 😅 

```suggestion
  getStore().dispatch(sendAnalytics(EVENTS.CODE_COPIED, {isTA}));
```